brightening-lambda.cpp

```diff
@@ -7,7 +7,7 @@ int main() {
 
 	if (image->imageSizeIsValid()) {
 		ImageBrightener brightener(image);
-		int attenuatedCount = brightener.BrightenWholeImage();
+		const int attenuatedCount = brightener.BrightenWholeImage();
```

Comment:
Good use of const to indicate that `attenuatedCount` does not change from here on.